bombast/__init__.py

```diff
@@ -135,6 +135,27 @@ def visit_ClassDef(self, node):
             return ClassDef(name, bases,
                             node.keywords, body, decorator_list)
 
+    def visit_JoinedStr(self, node):
+        new_node = None
+        for value in node.values:
+            if type(value) == Constant:
+                value = self.visit(value)
+            elif type(value) == FormattedValue:
+                value.value = self.visit(value.value)
+                value = JoinedStr(values=[value])
```

Comment:
Can you pull this out into `visit_FormattedValue`? Then, we can call `self.visit` on each `value` without explicitly checking the type.